torch/csrc/distributed/rpc/init.cpp

```diff
@@ -206,9 +223,13 @@ PyObject* rpc_init(PyObject* _unused, PyObject* noargs) {
               >>> rpc.rpc_sync("worker1", f, args=(rref,))
           )")
           .def(
-              py::init<const py::object&, const py::object&>(),
+              py::init<
+                  const py::object&,
+                  const py::object&,
+                  optional<std::vector<c10::DeviceIndex>>>(),
               py::arg("value"),
-              py::arg("type_hint") = py::none())
+              py::arg("type_hint") = py::none(),
```

Comment:
Looks yes. I will only do this for `devices` for now, as `type_hint` already allow position arguments since a few releases ago.

https://pybind11.readthedocs.io/en/stable/advanced/functions.html#keyword-only-arguments